src/KubernetesClient/Yaml.cs

```diff
@@ -105,38 +126,23 @@ public static List<object> LoadAllFromString(string content, Dictionary<string,
                 throw new ArgumentNullException(nameof(typeMap));
             }
 
-            var deserializer =
```

Comment:
I think the reason of `new` here is that the deserializer might not be thread safe

Reply:
seems I am wrong after we updated to 8.1.2
https://github.com/aaubry/YamlDotNet/blob/2e4e5bb402cafee22f36abd65423d60e60221b40/releases/8.1.0.md

should double confirm then